src/cincinnati/client.rs

```diff
@@ -34,6 +34,33 @@ pub struct Graph {
     pub edges: Vec<(u64, u64)>,
 }
 
+/// Error related to the graph service.
+#[derive(Clone, Debug, Fail, Deserialize, Serialize, PartialEq, Eq)]
+pub struct GraphError {
+    /// HTTP status code returned by the server.
+    #[serde(skip)]
+    pub(crate) status: Option<reqwest::StatusCode>,
```

Comment:
Minor: I'd call this `status_code` to make it more obvious why client-side failures have `None`.